src/psqlgraph/psql.py

```diff
@@ -106,7 +108,7 @@ def __init__(
 
         # Create driver engine
         self.engine = sqlalchemy.create_engine(
-            f"postgresql+psycopg2://{user}:{password}@{host}/{database}",
+            f"{ENGINE_SCHEME}://{user}:{password}@{host}/{database}",
```

Comment:
Might be good to pass the scheme or portions of it as a parameter that defaults to `psycopg2`

Reply:
Philosophical difference here. I would not set a default value but provide a useful exception if not defined.